#ifndef POHXI_CAMERA_TEST
#define POHXI_CAMERA_TEST
# include "ros/ros.h"
# include <iostream>
# include "phoxi_camera/ConnectCamera.h"
# include "phoxi_camera/SaveFrame.h"
# include "phoxi_camera/GetFrame.h"
# include "phoxi_camera/GetDeviceList.h"
# include "sensor_msgs/Image.h"
using namespace std;
using namespace ros;
#endif